package.json

```diff
@@ -57,7 +57,7 @@
     "nodemailer-mailgun-transport": "^1.4.0",
     "pg": "^4.4.1",
     "pg-hstore": "^2.3.2",
-    "sequelize": "^3.19.0",
+    "sequelize": "^6.0.0",
```

Comment:
I see `5.16.0` as the last release: https://github.com/sequelize/sequelize/releases
Should we use that version until there are release notes about 6?

Reply:
Definitely. The security advisory suggested @6.0.0, but from what I saw the issue is fixed from like 5.3.0 on, so I'll set this to the latest release.